.github/workflows/ocwm-reminders-tuesday.yml

```diff
@@ -0,0 +1,78 @@
+name: Send reminders to join the OCWM
+
+on:
+  schedule:
+    - cron: '0 15 1-7 * 5' #Runs at 15:00, between day 1 and 7 of the month on Friday 
+    - cron: '0 15 1-7 * 2' #Runs at 15:00, between day 1 and 7 of the month on Tuesday
+
```

Comment:
Here an issue might occur that Tuesday comes first before the Friday in upcoming month I have handled that below if the logic seems good to you tell me otherwise I can make appropriate changes in that